### ISSUE TITLE
Manual tests for Windows x64 0.13.0 Preview 6

### ISSUE THREAD

**Issue by alexwykoff** [brave/browser-laptop#6334]
## Installer

1. [ ] Check that installer is close to the size of last release.
2. [ ] Check signature: If OS Run `spctl --assess --verbose /Applications/Brave.app/` and make sure it returns `accepted`.  If Windows right click on the installer exe and go to Properties, go to the Digital Signatures tab and double click on the signature.  Make sure it says "The digital signature is OK" in the popup window.
3. [ ] Check Brave, electron, and libchromiumcontent version in About and make sure it is EXACTLY as expected.

## Per release specialty tests

- [ ] Geolocation not working #6331
- [ ] Changing language settings prompts to restart the browser #6416
- [ ] Changing language should prompt to restart #4777
- [ ] .exe file on Windows is 0KB, breaking the shortcut created on the desktop #6313
- [ ] Do not track only works after browser restart #6409
- [ ] opening a link in a new tab creates the new tab at the most left #6411
- [ ] default zoom level effects browser chrome UI and should not w/ C54 #6315
- [ ] Regression: about:brave doesn't show details #6406
- [ ] Get each version in a more safe way; if one field fails, it won't break the whole thing #6410
- [ ] Dead area in grab zone #5792
- [ ] Duplicated tabs should always go directly next to the opener tab #6390
- [ ] [Bug] Crash when use close other tabs #5872
- [ ] Maximized window has a White border around #6258
- [ ] Going back/forward button on about:preferences does not work #6343
- [ ] Makes notification labels, buttons behave more appropriately. #6418
- [ ] Labels and buttons in notification bar behave unpredictably #6417
- [ ] Fix copy version info tooltip #6392
- [ ] Setting min-width for buttons (l10n-friendly) #6384
- [ ] Setting min-width for buttons #6387
- [ ] Updated sanitization logic for separators #6374
- [ ] Pref change restart doesn't relaunch the browser #5938
- [ ] Buttons alignment on about:certerror and NoScript dialog #6317
- [ ] PDF doesn't load when opened but work by clicking on the link #6330
- [ ] Buttons inside #navigator are not working #6253
- [ ] Small fixes of buttons on about:certerror and .noScriptInfo #6324
- [ ] The about screen should show the latest SHA hash #5462
- [ ] hide-autofill-popup only when webcontent is focused #5959
- [ ] Feature: WebTorrent Integration #3256
- [ ] When Modal Overlays are layered, only bottom layer has a dark background (fix #4808) #4820
- [ ] Create default topSites for new tab page #5623
- [ ] Change hover of bookmark bar items #5668
- [ ] Add new "empty new tab" option in preferences (for what new tab does) #5690
- [ ] do not count a transaction-in-progress as a transaction... #5721
- [ ] Updates for URL bar and tab (Windows only) #5752
- [ ] Add user-friendly title for about:blank #5782
- [ ] Update findbar state onInput #5786
- [ ] Add tooltips to external links in the UI #5787
- [ ] Add context menu to reload button #5802
- [ ] minimum visits and minduration not persisting when changed #5824 
- [ ] Add default icon for url that has a broken file #5826
- [ ] Open bookmarks in new foreground tabs #5831
- [ ] Remove tab icons for about:newtab and about:blank #5845
- [ ] Clean up formatting for bitcoin wallet backup #5862
- [ ] Prevent duplicate separators for all context menu items #5869 
- [ ] Update newtab stats to have locale-aware formatting #5888
- [ ] should have been included in the previous PR... #5905
- [ ] Replace Payment History CSV export with Contribution Statement PDF (rebased) #5909 
- [ ] make button styles consistent #5916
- [ ] Torrent Viewer: change "Download" to "Save File" #5923
- [ ] initial check-in #5924
- [ ] Update forms.less to add hover effect #5954
- [ ] Add layer for topSites to avoid dupe icon/bg color #5984
- [ ] fixing contribution stmt PDF filename #5999
- [ ] Make addFunds open adjacent to Preferences tab #6060
- [ ] Private tab/Session tab icon needs right padding before text #6031
- [ ] these edits add 2 px to our tab height #6032
- [ ] input and select are not equal width #6035
- [ ] cleanup about page handling and fix missing document.l10n #6062
- [ ] Always show bookmarks folder title #6095
- [ ] Harden context menu icon handling and add getPathFromFileURI utility function #6112
- [ ] Adds 9 new pictures for use w/ about:newtab #6207
- [ ] Chromium54 #6247
- [ ] Import browser data switches misaligned #5863
- [ ] Add currency symbols where appropriate #6249
- [ ] Tweak on PinnedTopSite icon to avoid overflow in new tab page #6286
- [ ] Put back in handler for null/empty context menu detail. This is requi… #6250
- [ ] Properly align pinned tab icon #6301
- [ ] Reduce space for count/text inside timeSaved stat #5996
- [ ] Add WebTorrent support #5342 https://github.com/brave/browser-laptop/pull/5342#issuecomment-261687818
- [ ] Add Paste Without Formatting to context menu #6126
- [ ] Redo: Add copy to clipboard for about brave page (fixes #5790) #6298
- [ ] Button tweak prefs font nicer #6000
- [ ] Made every place of buttons on the navigation bar clickable #5750
- [ ] Stop about pages from being shown on topSites #5728
- [ ] Font-size of select element inside braveryControlGroup was set to 13px #5773
- [ ] Font rendering and "lang" attribute #2929
- [ ] Fixed font-weight on about:preferences#payments #6311
- [ ] Made the width of "Advanced Settings..." button on about:preferences#payments wider #6287
- [ ] Added ellipses to "View payment history" link #6073
- [ ] Replaced fa-times for close button on download bar with SVG button (redo #6071) #6153
- [ ] Made all place of Cubits/buybitcoinworldwide.com button clickable #6051
- [ ] Added margin-bottom to the transfer funds button on about:preferences#payments #6049
- [ ] Refactor about:preferences#payments followups #6047
- [ ] Refactor about:preferences#payments #6009
- [ ] Removed the hacks of the buttons on the navigation bar (for Windows) #5967
- [ ] Follow-up: Button styles consistent #6001
- [ ] Made the new tab button touch screen friendly #5927
- [ ] Aligned brave wallet keys #5808
- [ ] Removed "inlineButton" from Cancel button on Brave wallet recovery dialog #5805 
- [ ] Small changes of the icons on about pages #6138
- [ ] Aligned the bookmark icon on the navbar #5739
- [ ] Refactor buttons on about:preferences#payments and bookmark dialog([#5705](https://github.com/brave/browser-laptop/pull/5705))
- [ ] copy to clipboard with keyboard is broken in URL bar ([#6302](https://github.com/brave/browser-laptop/issues/6302))
- [ ] Browser is blurry if DPI is greater than 100% ([#5926](https://github.com/brave/browser-laptop/issues/5926))
- [ ] Pinned tab icon is not properly aligned ([#6300](https://github.com/brave/browser-laptop/issues/6300))
- [ ] mute tab is not working ([#6305](https://github.com/brave/browser-laptop/issues/6305))
- [ ] Clicking the button linked to Cubits opens a blank tab without loading the page ([#6152](https://github.com/brave/browser-laptop/issues/6152))
- [ ] Submit feedback under helpful hints opens about:blank page and not community webpage ([#6252](https://github.com/brave/browser-laptop/issues/6252))
- [ ] Regression : Alignment off for about:about ([#6183](https://github.com/brave/browser-laptop/issues/6183))
- [ ] about:extension doesn't contain Favicons of any of the installed extensions ([#6257](https://github.com/brave/browser-laptop/issues/6257))
- [ ] CMD + click on about pages are blank on new tab ([#5880](https://github.com/brave/browser-laptop/issues/5880))
- [ ] toLocale* methods not respecting locale ([#5879](https://github.com/brave/browser-laptop/issues/5879))
- [ ] Netflix extension is downloaded but still not being found by Brave ([#6278](https://github.com/brave/browser-laptop/issues/6278))
- [ ] about:newtab top site pins have the wrong z-index and are being hidden by top site icons ([#6281](https://github.com/brave/browser-laptop/issues/6281))
- [ ] Geolocation doesn't detect user location ([#5941](https://github.com/brave/browser-laptop/issues/5941))
- [ ] "Advanced Settings..." button on about:preferences#payments should be wider (l10n) ([#6285](https://github.com/brave/browser-laptop/issues/6285))
- [ ] Flash controls are not working ([#5883](https://github.com/brave/browser-laptop/issues/5883))
- [ ] Datalist element not supported (github drop down for releases page) ([#5213](https://github.com/brave/browser-laptop/issues/5213))
- [ ] Add currency symbols where appropriate ([#3533](https://github.com/brave/browser-laptop/issues/3533))
- [ ] Menu will not disappear ([#6236](https://github.com/brave/browser-laptop/issues/6236))
- [ ] Amazon pages go black ([#5881](https://github.com/brave/browser-laptop/issues/5881))
- [x] Replace close button on downloadBar with SVG button ([#6070](https://github.com/brave/browser-laptop/issues/6070))
- [ ] Closing devtools creates a white blank window ([#5940](https://github.com/brave/browser-laptop/issues/5940))
- [ ] Bookmarks shows wrong folder name ([#4983](https://github.com/brave/browser-laptop/issues/4983))
- [ ] Broken favicon on ledger table ([#6235](https://github.com/brave/browser-laptop/issues/6235))
- [ ] Refactoring #bitcoinDashboard dialog on about:preferences#payments (l10n) ([#6007](https://github.com/brave/browser-laptop/issues/6007))
- [ ] receipt links should not be wrapped ([#6230](https://github.com/brave/browser-laptop/issues/6230))
- [ ] border-bottom of the table header of the payment history should be extended ([#6231](https://github.com/brave/browser-laptop/issues/6231))
- [ ] OK button should be larger on Payment History dialog ([#6232](https://github.com/brave/browser-laptop/issues/6232))
- [ ] Addfunds dialog needs a little margin ([#5799](https://github.com/brave/browser-laptop/issues/5799))
- [ ] border-radius of modal dialog in .paymentsContainer should be 8px ([#6223](https://github.com/brave/browser-laptop/issues/6223))
- [ ] Funds FAQ icon is underlined on hover ([#6083](https://github.com/brave/browser-laptop/issues/6083))
- [ ] The buttons on the footers of modal dialogs of payments need to be aligned to right ([#6219](https://github.com/brave/browser-laptop/issues/6219))
- [ ] Buttons on about:payments are thinner than those on other about pages ([#6216](https://github.com/brave/browser-laptop/issues/6216))
- [ ] the buttons on about:preferences#payments need more paddings ([#6081](https://github.com/brave/browser-laptop/issues/6081))
- [ ] fa-mobile should be aligned at the center ([#6222](https://github.com/brave/browser-laptop/issues/6222))
- [ ] "Your Payment History" should be orange ([#6229](https://github.com/brave/browser-laptop/issues/6229))
- [ ] verified icon on the ledger table should be outside of the anchor link to the site ([#6225](https://github.com/brave/browser-laptop/issues/6225))
- [ ] Recovery keys should not be line-wrapped ([#6220](https://github.com/brave/browser-laptop/issues/6220))
- [ ] Design payment history table ([#6202](https://github.com/brave/browser-laptop/issues/6202))
- [ ] Payment History popover probably needs a little more attention ([#3485](https://github.com/brave/browser-laptop/issues/3485))
- [ ] Place the footer on the contribution report properly ([#6061](https://github.com/brave/browser-laptop/issues/6061))
- [ ] Transfer funds button needs margin-bottom ([#6048](https://github.com/brave/browser-laptop/issues/6048))
- [ ] Design "hide sites if" options on Advanced Options Dialog ([#6201](https://github.com/brave/browser-laptop/issues/6201))
- [ ] Title bar should not have strange spacing in general (on about:preferences#payments) ([#6221](https://github.com/brave/browser-laptop/issues/6221))
- [ ] Autofill popup will not be dismissed ([#6233](https://github.com/brave/browser-laptop/issues/6233))
- [ ] Tabs.create isn't actually creating a tab ([#6159](https://github.com/brave/browser-laptop/issues/6159))
- [ ] about:newtab - more beautiful backgrounds ([#6206](https://github.com/brave/browser-laptop/issues/6206))
- [ ] hash 'rmd160' not in chromium54/node/... build ([#6146](https://github.com/brave/browser-laptop/issues/6146))
- [ ] Chromium54 brave password causes crashes on for instance hacker news ([#6099](https://github.com/brave/browser-laptop/issues/6099))
- [ ] Window.onbeforeunload not working in Brave ([#4079](https://github.com/brave/browser-laptop/issues/4079))
- [ ] Add "hide sites if" options to Advanced Options Dialog ([#4681](https://github.com/brave/browser-laptop/issues/4681))
- [ ] built-in password manager should respect preference ([#6169](https://github.com/brave/browser-laptop/issues/6169))
- [ ] 1Password Should Auto-Submit on Fill ([#5875](https://github.com/brave/browser-laptop/issues/5875))
- [ ] Chromium54 - Keyboard shortcuts are not working on URL bar ([#5795](https://github.com/brave/browser-laptop/issues/5795))
- [ ] Clicking print/save on about:preferences#payments causes a crash on cr54 ([#6150](https://github.com/brave/browser-laptop/issues/6150))
- [ ] IPC is failing on C54 in renderer processes ([#6151](https://github.com/brave/browser-laptop/issues/6151))
- [x] Pressing escape doesn't clear url input in C54 ([#6148](https://github.com/brave/browser-laptop/issues/6148))
- [x] Fa-icons on about pages are not aligned consistently ([#6137](https://github.com/brave/browser-laptop/issues/6137))
- [ ] webview.focus stopped working in C54 ([#6122](https://github.com/brave/browser-laptop/issues/6122))
- [ ] Support dock menus ([#3024](https://github.com/brave/browser-laptop/issues/3024))
- [x] View payment history link needs padding ([#4867](https://github.com/brave/browser-laptop/issues/4867))
- [x] addfunds tab not properly placed ([#5913](https://github.com/brave/browser-laptop/issues/5913))
- [x] Private tab/Session tab icon needs right padding before text ([#5936](https://github.com/brave/browser-laptop/issues/5936))
- [ ] Remaining Linux crashes and nits for 54 ([#6091](https://github.com/brave/browser-laptop/issues/6091))
- [x] Clicking link from email will sometimes will do a search instead of directly following ([#5911](https://github.com/brave/browser-laptop/issues/5911))
- [x] [Feature Request] Bookmark text only for folders ([#6078](https://github.com/brave/browser-laptop/issues/6078))
- [x] Brave Wallet creation fails on Windows ([#5930](https://github.com/brave/browser-laptop/issues/5930))
- [x] Yandex favicon is 404 on about:preferences#search ([#6082](https://github.com/brave/browser-laptop/issues/6082))
- [ ] [HackerOne] issue #175927 ([#4973](https://github.com/brave/browser-laptop/issues/4973))
- [x] Close all tabs causes crash ([#5946](https://github.com/brave/browser-laptop/issues/5946))
- [x] Not all place of Cubits/buybitcoinworldwide.com button is clickable ([#6050](https://github.com/brave/browser-laptop/issues/6050))
- [x] Context menu cut and paste does not change findbar state ([#5753](https://github.com/brave/browser-laptop/issues/5753))
- [x] background color of modal window on ledger advanced settings gets darker by clicking backup wallet button ([#4808](https://github.com/brave/browser-laptop/issues/4808))
- [x] Bookmark dialog looks too skinny ([#6036](https://github.com/brave/browser-laptop/issues/6036))
- [ ] document.l10n is sometimes undefined ([#6012](https://github.com/brave/browser-laptop/issues/6012))
- [x] width of input and select elements on new bookmark folder form is not equal ([#5192](https://github.com/brave/browser-laptop/issues/5192))
- [x] fix bookmark hanger to have 250px min width, and right-align buttons ([#6004](https://github.com/brave/browser-laptop/issues/6004))
- [x] adjust fonts in prefs and other about pages to use helv neue on mac, and arial on win ([#6003](https://github.com/brave/browser-laptop/issues/6003))
- [x] Gray footer which contains nothing on the QR code overlay window on about:preferences#payments ([#5785](https://github.com/brave/browser-laptop/issues/5785))
- [ ] Many Payments Panel tests failing ([#5951](https://github.com/brave/browser-laptop/issues/5951))
- [x] The row of the table on about:preferences#payments is not completely aligned ([#5962](https://github.com/brave/browser-laptop/issues/5962))
- [x] Clicking on Shields down on C54 crashes the browser ([#6034](https://github.com/brave/browser-laptop/issues/6034))
- [x] increase tabs height by 3 px ([#5978](https://github.com/brave/browser-laptop/issues/5978))
- [x] Links to external URLs in the UI should have tooltips ([#5759](https://github.com/brave/browser-laptop/issues/5759))
- [ ] [hackerone] 188086 ([#6011](https://github.com/brave/browser-laptop/issues/6011))
- [x] Focus on tabs keeps "bouncing" back to previous tab ([#5730](https://github.com/brave/browser-laptop/issues/5730))
- [x] Enable Pocket/Password managers doesn't enable the extension ([#5934](https://github.com/brave/browser-laptop/issues/5934))
- [x] Unable to delete history entries by clicking on clear browsing data ([#5933](https://github.com/brave/browser-laptop/issues/5933))
- [ ] Import Bookmark doesn't have any browsers listed except for Chrome ([#5931](https://github.com/brave/browser-laptop/issues/5931))
- [ ] 0.13.0 Preview1 / Chromium 54 tracking issue ([#5877](https://github.com/brave/browser-laptop/issues/5877))
- [x] Extension downloading not working for C54 ([#5945](https://github.com/brave/browser-laptop/issues/5945))
- [ ] Refactor download handling so it works with node removal from window renderer ([#5947](https://github.com/brave/browser-laptop/issues/5947))
- [ ] Updated to 0.13.0 and payment doesn't process ([#5974](https://github.com/brave/browser-laptop/issues/5974))
- [x] Add drop down options to reload button ([#5796](https://github.com/brave/browser-laptop/issues/5796))
- [x] Autofill suggestion doesn't complete autofill ([#5939](https://github.com/brave/browser-laptop/issues/5939))
- [x] Favicon of trakt.tv on new tab page ([#5868](https://github.com/brave/browser-laptop/issues/5868))
- [ ] [hackerone] 182306 ([#5672](https://github.com/brave/browser-laptop/issues/5672))
- [x] Make button styles consistent ([#5801](https://github.com/brave/browser-laptop/issues/5801))
- [x] Payment Button mouse over barely noticeable ([#4708](https://github.com/brave/browser-laptop/issues/4708))
- [ ] [hackerone] 187542 ([#5980](https://github.com/brave/browser-laptop/issues/5980))
- [ ] HTML5 Battery Status API ([#1885](https://github.com/brave/browser-laptop/issues/1885))
- [x] URL bar issues on Chromium54 build(0.13.0 preview2) ([#5928](https://github.com/brave/browser-laptop/issues/5928))
- [x] New tab button is not touch screen friendly ([#5925](https://github.com/brave/browser-laptop/issues/5925))
- [ ] Upgrade to Chromium v54 on or around Oct 18th, 2016 (1 week before potentially) ([#3679](https://github.com/brave/browser-laptop/issues/3679))
- [ ] "Remove Bookmark" button hard to read on hover ([#5894](https://github.com/brave/browser-laptop/issues/5894))
- [ ] Edit Bookmark close icon needs hover color ([#5953](https://github.com/brave/browser-laptop/issues/5953))
- [x] Coinbase should not be shown in payment modal when bitcoin source is not Coinbase ([#5788](https://github.com/brave/browser-laptop/issues/5788))
- [ ] Query wallet balance less ([#4114](https://github.com/brave/browser-laptop/issues/4114))
- [x] Opening a new tab shows Uncaught TypeError in browser console ([#5935](https://github.com/brave/browser-laptop/issues/5935))
- [x] Brave doesn't create a roaming profile folder ([#5929](https://github.com/brave/browser-laptop/issues/5929))
- [ ] Change download icon to save after torrent is downloaded ([#5815](https://github.com/brave/browser-laptop/issues/5815))
- [ ] LSOpenURLsWithRole() failed with error -10810 on the packaged build ([#5849](https://github.com/brave/browser-laptop/issues/5849))
- [ ] base64Encode required but never used ([#5914](https://github.com/brave/browser-laptop/issues/5914))
- [ ] Replace Payment History CSV export with Contribution Statement PDFs ([#4769](https://github.com/brave/browser-laptop/issues/4769))
- [x] Bookmark icon is not properly aligned with URL Bar ([#5632](https://github.com/brave/browser-laptop/issues/5632))
- [x] Plugin settings info icons are misaligned ([#5760](https://github.com/brave/browser-laptop/issues/5760))
- [ ] add comma/decimals to newtab brave info tallies ([#5827](https://github.com/brave/browser-laptop/issues/5827))
- [ ] About pages right click context menu has an extra separator ([#5765](https://github.com/brave/browser-laptop/issues/5765))
- [x] Set 2 visits on minimum visit for publisher relevancy and automatically back to 5 ([#4822](https://github.com/brave/browser-laptop/issues/4822))
- [x] Import browser data switches misaligned ([#5860](https://github.com/brave/browser-laptop/issues/5860))
- [x] contents of ledger recovery key text file should be set to the spec ([#4807](https://github.com/brave/browser-laptop/issues/4807))
- [ ] Favicons Only Behavior Now Leaves Blanks when there is no favicon ([#3648](https://github.com/brave/browser-laptop/issues/3648))
- [x] bookmark is invisible if favicon.ico is not found ([#2075](https://github.com/brave/browser-laptop/issues/2075))
- [ ] Favicons need onerror handler (as seen in about:newtab) ([#5455](https://github.com/brave/browser-laptop/issues/5455))
- [x] Dropdown not shown completely when right click/long press on newtab button ([#5763](https://github.com/brave/browser-laptop/issues/5763))
- [x] Newtab should not have an empty favicon ([#5776](https://github.com/brave/browser-laptop/issues/5776))
- [x] Font-size of select element inside braveryControlGroup should be set to 13px ([#5772](https://github.com/brave/browser-laptop/issues/5772))
- [ ] SyntaxError: Unexpected token: name (WebTorrentRemoteClient) ([#5818](https://github.com/brave/browser-laptop/issues/5818))
- [x] Middle click on a bookmark link opens tab in the background ([#3207](https://github.com/brave/browser-laptop/issues/3207))
- [ ] Ad control for first time opened private tab doesn't work properly ([#5701](https://github.com/brave/browser-laptop/issues/5701))
- [x] keyContainer on Brave wallet backup dialog is not aligned ([#5807](https://github.com/brave/browser-laptop/issues/5807))
- [x] Magnifying glass icon shown for pdf documents opened in Brave ([#5774](https://github.com/brave/browser-laptop/issues/5774))
- [x] Buttons not aligned properly in payments advance settings in different language ([#5771](https://github.com/brave/browser-laptop/issues/5771))
- [x] l10n of ledger advanced settings modal window ([#4786](https://github.com/brave/browser-laptop/issues/4786))
- [x] Create "Empty New Tab" option in General Settings ([#5682](https://github.com/brave/browser-laptop/issues/5682))
- [ ] Ledger notifications for adding funds should only be shown if brave payments is switched on ([#5441](https://github.com/brave/browser-laptop/issues/5441))
- [ ] Monthly ledger payment failed ([#5371](https://github.com/brave/browser-laptop/issues/5371))
- [x] Ledger does not track windows created before the wallet ([#4274](https://github.com/brave/browser-laptop/issues/4274))
- [x] about:blank shows chrome-extension in navigation button history ([#5764](https://github.com/brave/browser-laptop/issues/5764))
- [x] newtab - Have default top sites available ([#5594](https://github.com/brave/browser-laptop/issues/5594))
- [ ] transition of reload button ([#5664](https://github.com/brave/browser-laptop/issues/5664))
- [x] Hover background of bookmark bar items is hard to see ([#5665](https://github.com/brave/browser-laptop/issues/5665))
- [x] borders and corners of each button on navbar are not clickable ([#5679](https://github.com/brave/browser-laptop/issues/5679))
- [x] add windows specific font CSS for URL text and tab text ([#5624](https://github.com/brave/browser-laptop/issues/5624))
- [x] about:newtab appears on the top sites list ([#5638](https://github.com/brave/browser-laptop/issues/5638))
- [x] Findbar text selection ([#5744](https://github.com/brave/browser-laptop/issues/5744))

## Ledger
1. [ ] Create a wallet with a value other than $5 selected in the monthly budget dropdown. Click on the 'Add Funds' button and check that Coinbase transactions are blocked.
2. [ ] Remove all `ledger-*.json` files from `~/Library/Application\ Support/Brave/`. Go to the Payments tab in about:preferences, enable payments, click on `create wallet`. Check that the `add funds` button appears after a wallet is created.
3. [ ] Click on `add funds` and verify that adding funds through Coinbase increases the account balance.
4. [ ] Repeat the step above but add funds by scanning the QR code in a mobile bitcoin app instead of through Coinbase.
5. [ ] Visit nytimes.com for a few seconds and make sure it shows up in the Payments table.
6. [ ] Go to https://jsfiddle.net/LnwtLckc/5/ and click the register button. In the Payments tab, click `add funds`. Verify that the `transfer funds` button is visible and that clicking on `transfer funds` opens a jsfiddle URL in a new tab.
7. [ ] Go to https://jsfiddle.net/LnwtLckc/5/ and click `unregister`. Verify that the `transfer funds` button no longer appears in the `add funds` modal.
8. [ ] Check that disabling payments and enabling them again does not lose state.

## Data

1. [ ] Make sure that data from the last version appears in the new version OK.
2. [ ] Test that the previous version's cookies are preserved in the next version.

## About pages

1. [ ] Test that about:adblock loads
2. [ ] Test that about:autofill loads
3. [ ] Test that about:bookmarks loads bookmarks
4. [ ] Test that about:downloads loads downloads
5. [ ] Test that about:extensions loads
6. [ ] Test that about:history loads history
7. [ ] Test that about:passwords loads
8. [ ] Test that about:styles loads
9. [ ] Test that about:preferences changing a preference takes effect right away
10. [ ] Test that about:preferences language change takes effect on re-start
11. [ ] Test that about.com without the http protocol is loaded from URL bar

## Bookmarks

1. [ ] Test that creating a bookmark on the bookmarks toolbar works
2. [ ] Test that creating a bookmark folder on the bookmarks toolbar works
3. [ ] Test that moving a bookmark into a folder by drag and drop on the bookmarks folder works
4. [ ] Test that clicking a bookmark in the toolbar loads the bookmark.
5. [ ] Test that clicking a bookmark in a bookmark toolbar folder loads the bookmark.

## Context menus
1. [ ] Make sure context menu items in the URL bar work
2. [ ] Make sure context menu items on content work with no selected text.
3. [ ] Make sure context menu items on content work with selected text.
4. [ ] Make sure context menu items on content work inside an editable control (input, textarea, or contenteditable).

## Find on page

1. [ ] Ensure search box is shown with shortcut
2. [ ] Test successful find
3. [ ] Test forward and backward find navigation
4. [ ] Test failed find shows 0 results
5. [ ] Test match case find

## Geolocation

1. [ ] Check that https://developer.mozilla.org/en-US/docs/Web/API/Geolocation/Using_geolocation works

## Site hacks

1. [ ] Test https://www.twitch.tv/adobe sub-page loads a video and you can play it

## Downloads

1. [ ] Test downloading a file works and that all actions on the download item works.

## Fullscreen

1. [ ] Test that entering full screen window works View -> Toggle Full Screen. And exit back (Not Esc).
2. [ ] Test that entering HTML5 full screen works. And Esc to go back. (youtube.com)

## Tabs and Pinning

1. [ ] Test that tabs are pinnable
2. [ ] Test that tabs are unpinnable
3. [ ] Test that tabs are draggable to same tabset
4. [ ] Test that tabs are draggable to alternate tabset

## Zoom

1. [ ] Test zoom in / out shortcut works
2. [ ] Test hamburger menu zooms.
3. [ ] Test zoom saved when you close the browser and restore on a single site.
4. [ ] Test zoom saved when you navigate within a single origin site.
5. [ ] Test that navigating to a different origin resets the zoom

## Bravery settings

1. [ ] Check that HTTPS Everywhere works by loading http://www.apple.com
2. [ ] Turning HTTPS Everywhere off and shields off both disable the redirect to https://www.apple.com
3. [ ] Check that ad replacement works on http://slashdot.org
4. [ ] Check that toggling to blocking and allow ads works as expected.
5. [ ] Test that clicking through a cert error in https://badssl.com/ works.
6. [ ] Test that Safe Browsing works (http://downloadme.org/)
7. [ ] Turning Safe Browsing off and shields off both disable safe browsing for http://downloadme.org/.
8. [ ] Visit https://brianbondy.com/ and then turn on script blocking, nothing should load. Allow it from the script blocking UI in the URL bar and it should work.
9. [ ] Test that about:preferences default Bravery settings take effect on pages with no site settings.
10. [ ] Test that turning on fingerprinting protection in about:preferences shows 3 fingerprints blocked at https://jsfiddle.net/bkf50r8v/13/. Test that turning it off in the Bravery menu shows 0 fingerprints blocked.
11. [ ] Test that 3rd party storage results are blank at https://jsfiddle.net/7ke9r14a/9/ when 3rd party cookies are blocked and not blank when 3rd party cookies are unblocked.
12. [ ] Test that audio fingerprint is blocked at https://audiofingerprint.openwpm.com/ when fingerprinting protection is on.

## Content tests

1. [ ] Go to https://brianbondy.com/ and click on the twitter icon on the top right. Test that context menus work in the new twitter tab.
2. [ ] Load twitter and click on a tweet so the popup div shows.   Click to dismiss and repeat with another div. Make sure it shows.
3. [ ] Go to http://www.bennish.net/web-notifications.html and test that clicking on 'Show' pops up a notification asking for permission. Make sure that clicking 'Deny' leads to no notifications being shown.
4. [ ] Go to https://trac.torproject.org/projects/tor/login and make sure that the password can be saved. Make sure the saved password shows up in `about:passwords`. Then reload https://trac.torproject.org/projects/tor/login and make sure the password is autofilled.
5. [ ] Open a github issue and type some misspellings, make sure they are underlined.
6. [ ] Make sure that right clicking on a word with suggestions gives a suggestion and that clicking on the suggestion replaces the text.
7. [ ] Make sure that Command + Click (Control + Click on Windows, Control + Click on Ubuntu) on a link opens a new tab but does NOT switch to it.  Click on it and make sure it is already loaded.
8. [ ] Open an email on http://mail.google.com/ or inbox.google.com and click on a link. Make sure it works.
9. [ ] Test that PDF is loaded at http://www.orimi.com/pdf-test.pdf
10. [ ] Test that https://mixed-script.badssl.com/ shows up as grey not red (no mixed content scripts are run).

## Flash tests
1. [ ] Turn on Flash in about:preferences#security. Test that clicking on 'Install Flash' banner on myspace.com shows a notification to allow Flash and that the banner disappears when 'Allow' is clicked.
2. [ ] Test that flash placeholder appears on http://www.y8.com/games/superfighters

## Autofill tests
1. [ ] Test that autofill works on http://www.roboform.com/filling-test-all-fields

## Session storage

Do not forget to make a backup of your entire `~/Library/Application\ Support/Brave` folder.

1. [ ] Temporarily move away your `~/Library/Application\ Support/Brave/session-store-1` and test that clean session storage works. (`%appdata%\Brave in Windows`, `./config/brave` in Ubuntu)
2. [ ] Test that windows and tabs restore when closed, including active tab.
3. [ ] Move away your entire `~/Library/Application\ Support/Brave` folder (`%appdata%\Brave in Windows`, `./config/brave` in Ubuntu)

## Cookie and Cache

1. [ ] Make a backup of your profile, turn on all clearing in preferences and shut down.  Make sure when you bring the browser back up everything is gone that is specified.
2. [ ] Go to http://samy.pl/evercookie/ and set an evercookie. Check that going to prefs, clearing site data and cache, and going back to the Evercookie site does not remember the old evercookie value.

## Update tests

1. [ ] Test that updating using `BRAVE_UPDATE_VERSION=0.8.3` env variable works correctly.

**Comment by bbondy** [comment]
Superseded by preview8 issues
